quickevent/doc/jednotny_format_vysledku_csos.txt

```diff
@@ -0,0 +1,33 @@
+Jednotný formát výsledků pro výpočet žebříčků, Rankingu a pro Internetovou stránku sekce OB:
```

Comment:
Tomuhle zrovna moc nerozumim, le nebyl by lepsi format pro dokumentaci MarkDown? Dokonce mi to TXT pripomina vizualne spis `md` :)

Reply:
Mame v QE export jednotneho formatu vysledku CSOS. A uz je docela problem najit jeho popis. Tak jsem popis pridal. Je to primo popis jez byl kdysi soucasti oficialnich dokumentu CSOS. 
Az bude export celkove odstranen z QE, lze smazat i tento popis.

Reply:
JJ, to je dopbre, ze jso ho pridal, spis jsem si rikal, ze v MD by to bylo citelnejsi, ale netrvam na to. To ,ze tomu nerozumim, znamenalo, ze tomu fakt nerozumim, ne ze nerozumim tomu, proc jsi to pridal :)